boilerplate/app/app.tsx

```diff
@@ -12,17 +12,29 @@
 import "./i18n"
 import "./utils/ignore-warnings"
 import { useFonts } from "expo-font"
-import React, { useEffect, useState } from "react"
+import React from "react"
 import { initialWindowMetrics, SafeAreaProvider } from "react-native-safe-area-context"
-import { RootStore, RootStoreProvider, setupRootStore } from "./models"
+import { useInitialRootStore } from "./models"
 import { AppNavigator, useNavigationPersistence } from "./navigators"
 import { ErrorBoundary } from "./screens/ErrorScreen/ErrorBoundary"
 import * as storage from "./utils/storage"
 import { customFontsToLoad } from "./theme"
+import { setupReactotron } from "./services/reactotron"
 
-// This puts screens in a native ViewController or Activity. If you want fully native
-// stack navigation, use `createNativeStackNavigator` in place of `createStackNavigator`:
-// https://github.com/kmagiera/react-native-screens#using-native-stack-navigator
+// Set up Reactotron, which is a free desktop app for inspecting and debugging
+// React Native apps. Learn more here: https://github.com/infinitered/reactotron
+setupReactotron({
+  // clear the Reactotron window when the app loads/reloads
+  clearOnLoad: true,
+  // generally going to be localhost
+  host: "localhost",
+  // Reactotron can moniter AsyncStorage for you
+  useAsyncStorage: true,
+  // log the initial restored state from AsyncStorage
+  logInitialState: true,
+  // log out any snapshots as they happen (this is useful for debugging but slow)
+  logSnapshots: false,
+})
```

Comment:
Expose the Reactotron config right here in app.tsx, making it easier to configure. Removed the previous note which applied to useScreens which is now gone.